mozilla/addons-server
# Hold actions on the back of decisions, rather than process them, if content is high profile

Fixes: mozilla/addons#15074



### Description

This patch changes decision handling from Cinder, to check if the decision/content is high profile, and if so to not carry out the action - just log the action that should have happened.

### Context

This is the first part of larger epic - further work will expose the held actions and allow them to be carried out (or not).  No decisions are coming from Cinder at this time so we're pretty safe to land it half finished ... though it could be argued it should be behind a waffle switch :shrug: 

### Testing

Bit of a pain to test fully, because it relies on having your cinder auth enabled for the report, and manually applying webhook payloads. But roughly:
- find a promoted (recommended, line, spotlight, or notable) add-on locally
  - note: the add-on must be actively promoted (i.e. has at least one version approved for that Promoted class)
- report that add-on to cinder under a category that is handled in cinder (e.g. legal reason) 
- in cinder stage, make a decision to disable on that add-on and copy the payload json from the admin section
- apply the payload via curl, etc (easier if you comment out `@permission_classes((CinderInboundPermission,))`!)
- go to reviewer review page for that add-on and see add-on is not disabled, but that the disable was attempted _is_ displayed in the version history

### Checklist
